packages/grpc-js/test/test-resolver.ts

```diff
@@ -460,7 +460,7 @@ describe('Name Resolver', () => {
             }
           },
         },
-        {}
+        { 'grpc.dns_min_time_between_resolutions_ms': 2000 }
```

Comment:
This shouldn't be necessary. This test expects DNS resolution to fail, and this option only takes effect if it succeeds.

Reply:
For this test I am getting error without this value. I assume it allow us to "retry" resolution faster but probably it's another issue.

```
Uncaught AssertionError [ERR_ASSERTION]: resultCount 1 !== 2
      + expected - actual

      -1
      +2
      
      at Timeout._onTimeout (test/test-resolver.ts:477:16)
      at listOnTimeout (node:internal/timers:569:17)
      at processTimers (node:internal/timers:512:7)
```

Also getting timeouts on, should I increase the value? 

```
  Should resolve gRPC interop servers:
     Error: Timeout of 4000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/Users/george.kampitakis/Dev/grpc-node/packages/grpc-js/build/test/test-resolver.js)
      at listOnTimeout (node:internal/timers:569:17)
      at processTimers (node:internal/timers:512:7)
```